.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   py2_serverInstall_serverTest:
     docker:
       # This image uses the oldest version of many dependencies
-      - image: girder/girder_test_py2:latest
+      - image: girder/girder_test:latest-py2
```

Comment:
Could we tag the images consistently?  If one is `latest-3.5`, this one should be `latest-2.7` (or the other should be `latest-py3`).

Reply:
They will be consistent when this branch is merged, but there is a final "WIP" commit on this branch, which is necessary to QA it (since otherwise this would run against the old Docker image fromt master).

The actual image names will be `girder/girder_test:latest-py2` and `girder/girder_test:latest-py3`.